main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
To loop 10 times, the loop should run while loop < 11 or, alternatively, loop <= 10.